### DISCUSSION TITLE
trying to look for obscure game, but doesn't exist in the database

### DISCUSSION THREAD

**Discussion by spaghettic** [mamedev/discussions#134] [Q&A] [answered]
i was trying to look for the obscure games like chill (psx eur, saturn version is there but not psx, and none of the three psx versions) and battle konchuuden (psx japan, no batkonchuden or anything similar found) do you guys have to verify if your disk dumps are good to get chds or do you just get the more recognizable ones out of the way first?

**Comment by angelosa** [comment, marked as answer]
`psx.xml` is incredibly incomplete, and we accept redump.org submissions about them as long as they are done in small batches (20-30 discs per PR).
http://redump.org/disc/6110/

**Reply by spaghettic** [reply]
Good to know, realized the hidden palace retail prototype build wasn't in redump, oops
